.php_cs

```diff
@@ -1,29 +1,28 @@
 #!/usr/bin/env php
 <?php
 
-$finder = Symfony\CS\Finder::create()
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude('app/config')
-    ->exclude('app/data')
-    ->exclude('app/Resources')
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
     ->exclude('var')
     ->exclude('vendor')
-    ->exclude('web/bundles')
-    ->exclude('web/css')
-    ->exclude('web/fonts')
-    ->exclude('web/js')
     ->notPath('web/config.php')
-    ->in(__DIR__)
+    // Doctrine fixtures are excluded to avoid these PHP-CS-Fixer errors:
+    // "Files were not fixed due to errors reported during linting before fixing"
+    ->notPath('DataFixtures')
 ;
 
-return Symfony\CS\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
-    ->fixers([
-        '-psr0', // Ignore Tests\ namespace prefix mismatch with tests/ directory
-        'ordered_use',
-        'phpdoc_order',
-        'short_array_syntax',
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'no_useless_else' => true,
+        'no_useless_return' => true,
+        'ordered_imports' => true,
+        'phpdoc_order' => true,
+        'php_unit_strict' => true,
+        'strict_comparison' => true,
```

Comment:
I am totally for this rule, but it's not from Symfony ruleset. Why include it in demo ? That would mean anyone would include here anything with time passing, right ? I would either keep Symfony rulesets only or propose those changes to be included in the ruleset in the first place.

Problematic parts are rules like `ordered_imports`, which Symfony wants to follow in new code, but don't want to apply it for whole codebase to avoid merging conflicts